flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/benchmark/StreamNetworkBenchmarkEnvironment.java

```diff
@@ -115,7 +115,8 @@ public void setUp(
 			int channels,
 			boolean localMode,
 			int senderBufferPoolSize,
-			int receiverBufferPoolSize) throws Exception {
+			int receiverBufferPoolSize,
+			Configuration config) throws Exception {
```

Comment:
How about overloading this method instead with a variant that also accepts a configuration? Then it would be easier to create benchmarks across different flink versions.

Reply:
sure, why not (I'm backporting this patch to Flink 1.5+ anyway, but not all tags)

-> did the same for the other `setUp()` methods